docs/CSBR.md

```diff
@@ -1370,35 +1962,49 @@ For Code Signing Certificates, Signing Services SHALL protect Subscriber Private
 
 Techniques that MUST be used to satisfy this requirement include:
 
-1.  Use of an Hardware Crypto Module, verified by means of a FIPS or Common Criteria certificate; or
-2.  A cloud-based key generation and protection solution with the following requirements:
-    a.  Key creation, storage, and usage of Private Key must remain within the security boundaries of the cloud solution’s Hardware Crypto Module that conforms to the specified requirements;
-    b.  Subscription at the level that manages the Private Key must be configured to log all access, operations, and configuration changes on the resources securing the Private Key.
+1. Use of an Hardware Crypto Module, verified by means of a FIPS or Common Criteria certificate; or
+2. A cloud-based key generation and protection solution with the following requirements:
+   1. Key creation, storage, and usage of Private Key must remain within the security boundaries of the cloud solution’s Hardware Crypto Module that conforms to the specified requirements;
+   2. Subscription at the level that manages the Private Key must be configured to log all access, operations, and configuration changes on the resources securing the Private Key.
 
 #### 6.2.7.4 Subscriber Private Key protection and verification 
 
 ##### 6.2.7.4.1 Subscriber Private Key protection
 
-Subscriber Private Keys for Code Signing Certificates SHALL be protected per the following requirements.
+For Non-EV Code Signing Certificates issued prior to June 1, 2023, the CA MUST obtain a representation from the Subscriber that the Subscriber will use one of the following options to generate and protect their Code Signing Certificate Private Keys:

```

Comment:
Consider removing all of this

Reply:
Re-use period is still in effect for those (825 days) so I am considering leaving as-is. Perhaps we can update in a future cleanup ballot. Any objections?

Reply:
I don't think the text is needed anymore since the then-current CSBRs are applicable to previous issuance and changes here have no bearing. That being said, I don't object to deferring this to a later ballot. I was only hoping to minimize the need of a future cleanup ballot by tackling now since we're already making changes in this section.

Reply:
I'm sure we'll have more to clean-up so punting for a separate ballot.